### DISCUSSION TITLE
Can't set rust_input to crate::api

### DISCUSSION THREAD

**Discussion by danijourdain** [fzyzcjy/flutter_rust_bridge#2107] [Q&A]
I'm trying to figure out how to scan external crates. My first problem is the rust_input field in flutter_rust_bridge.yaml.
When I set that using crate::api instead of rust/src/api/**/*.rs, I get this error:
`Error: Find zero rust input paths. (glob_pattern="[MY PATH]\\lib/src/api/**/*.rs")`

I'm on Windows 11, and I'm using a newly generated flutter_rust_bridge project.

**Comment by fzyzcjy** [comment]
Hi, could you please show full `flutter_rust_bridge.yaml` content?

**Reply by danijourdain** [reply]
Thanks for responding so fast! It turns out I had an old version of flutter rust bridge. Updating to v2.0.0-dev.40 fixed the issue. 

Here is the yaml file content if it helps anyone else: 

```yaml
rust_input: crate::api,zsi_audio
dart_output: lib/src/rust
# dump_all: true
```

**Reply by fzyzcjy** [reply]
Happy to see it is fixed and you are welcome!
